src/utils/openapi.ts

```diff
@@ -415,6 +419,29 @@ function humanizeRangeConstraint(
   return stringRange;
 }
 
+export function humanizeNumberRange(schema: OpenAPISchema): string | undefined {
+  const minimum =
+    typeof schema.exclusiveMinimum === 'number'
+      ? Math.min(schema.exclusiveMinimum, schema.minimum ?? Infinity)
+      : schema.minimum;
+  const maximum =
+    typeof schema.exclusiveMaximum === 'number'
+      ? Math.max(schema.exclusiveMaximum, schema.maximum ?? 0)
```

Comment:
It won't work for negative numbers. Would be good to add a test for this case maybe.
```suggestion
      ? Math.max(schema.exclusiveMaximum, schema.maximum ?? -Infinity)
```

Reply:
good point, thanks!